src/DynamoCoreWpf/Manipulation/DynamoManipulatorContext.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using Dynamo.Controls;
+
+namespace Dynamo.Wpf.Manipulation
+{
+    public class DynamoManipulatorContext
```

Comment:
Couldn't this be called simply `NodeManipulatorContext`? Or more simply, `ManipulatorContext`?